documentation/docs/scenarios/xsa-deploy/Readme.md

```diff
@@ -0,0 +1,105 @@
+# Build and Deploy SAP Fiori Applications on SAP HANA XS Advanced
+
+Build an application based on SAPUI5 or SAP Fiori with Jenkins and deploy the build result into an SAP Cloud Platform account in the Neo environment.
+
+## Prerequisites
+
+* [Docker environment](https://docs.docker.com/get-started/)
+* All artifacts refereneced during the build are available either on Service Market Place or via public repositories
+* You have set up Project “Piper”. See [guided tour](https://sap.github.io/jenkins-library/guidedtour/).
+* Docker image for xs deployment available. Due to legal reasons there is no pre-build docker image. How to create the docker image is explained [here](https://github.com/SAP/devops-docker-images/tree/master/xs-cli).
+### Project Prerequisites
+
+This scenario requires additional files in your project and in the execution environment on your Jenkins instance.
+
+On the project level, provide and adjust the following template:
+
+| File Name | Description | Position |
+|-----|-----|-----|
+| [`mta.yaml`](https://github.com/SAP/jenkins-library/blob/master/documentation/docs/scenarios/xsa-deploy/files/mta.yaml) | This file controls the behavior of the MTA toolset. | Place the `mta.yaml` file in your application root folder and adjust the values in brackets with your data. |
+
```

Comment:
As an idea I would just refer to the XS advanced developer quick start guide. https://help.sap.com/viewer/400066065a1b46cf91df0ab436404ddc/2.0.04/en-US/7f681c32c2a34735ad85e4ab403f8c26.html

It describes the entire project structure and the needed files.